ipmiMgrApp/Db/Makefile

```diff
@@ -10,7 +10,7 @@ include $(TOP)/configure/CONFIG
 #----------------------------------------------------
 # Create and install (or just install) into <top>/db
 # databases, templates, substitutions like this
-
+USR_DBFLAGS += -I$(IPMICOMM)/Db
```

Comment:
This should be a standalone commit, referencing the commit it fixes. Also mention that it can be removed as soon as a new release comes up.

Reply:
I don't like very much having a commit hash in the header line. A commonly used pattern is to add a trailer specifying the commit that is fixed. For instance, the message could be something like:

```
Search for database files in ipmiComm Db

We use some databases from the ipmiComm module that are not available in
the `db` directory, only in the `Db`. This will no longer be true as
soon as a new version containing the export [1] is released in
epics-in-docker.

Fixes: 14a436eaad83 (Change the locally defined Db's to use aliases from the ipmiComm db's)

[1]: https://github.com/slac-epics-modules/ipmiComm/commit/5e1939de5b14e49cb0c2e79f3df6ee8b0a94704c
```

Adding the message right after the hash is [a practice highly encouraged by Linux kernel community][1] and also commonly seen in Git project itself.

Also note that in my message suggestion, I've added enough information for someone to evaluate if the change should be reverted. IMO, that's better than having it loosely defined.

[1]: https://www.kernel.org/doc/html/v6.10/process/submitting-patches.html#describe-your-changes